WooCommerce/Classes/Authentication/Store Creation/Profiler/Features/StoreCreationFeaturesQuestionViewModel.swift

```diff
@@ -0,0 +1,74 @@
+import Combine
+import Foundation
+
+/// Necessary data from the answer of the store creation features question.
+struct StoreCreationFeaturesAnswer: Equatable {
+    /// Display name of the selected feature.
+    let name: String
+    /// Raw value of the feature to be sent to the backend.
+    let value: String
+}
+
+/// View model for `StoreCreationFeaturesQuestionView`, an optional profiler question about features in the store creation flow.
+@MainActor
+final class StoreCreationFeaturesQuestionViewModel: StoreCreationProfilerQuestionViewModel, ObservableObject {
```

Comment:
Nit: It looks to me that this view model and its view are very similar to the ones for challenges. Can we reuse the code and config only the question type, header, title, and subtitle for these two screens?

Reply:
I'm ok with this duplication (hence the approval). Just left my thought in case it's feasible or worthwhile 😄